### ISSUE TITLE
Call to undefined method Plank\Metable\Meta::getAttributesForInsert() 

### ISSUE THREAD

**Issue by matuskm** [plank/laravel-metable#62]
Hello,

I have issues with insert array meta - **`setManyMeta()`**.

I have:
```
$inorder->setManyMeta([
            'inorder_price' => $request->price,
            'customer_order' => $request->customer_order,
        ]);
```

and alert is:
```
BadMethodCallException
Call to undefined method Plank\Metable\Meta::getAttributesForInsert() 
```

![obrázok](https://user-images.githubusercontent.com/44832892/135715907-8d1d2fd4-1085-4d49-9dd9-3266e636a77a.png)

Please, help me :)

**Comment by frasmage** [comment]
Hi there,

What version of label are you using? Might be a new method. I can add a
version check.

**Comment by matuskm** [comment]
Do you think Laravel version? The laravel version is 8.0

**Comment by matuskm** [comment]
Do you mean Laravel version? Laravel version is 8.0

**Comment by frasmage** [comment]
Fixed in 5.0.2
